src/lib/route-broadcaster.js

```diff
@@ -65,14 +65,21 @@ class RouteBroadcaster {
   }
 
   broadcast () {
-    const adjacentConnectors = Object.keys(this.peers)
+    const adjacentLedgers = Object.keys(this.peersByLedger)
     const routes = this.routingTables.toJSON(SIMPLIFY_POINTS)
-    return Promise.all(adjacentConnectors.map((adjacentConnector) => {
-      log.info('broadcasting ' + routes.length + ' routes to ' + adjacentConnector)
-      const prefix = this.peers[adjacentConnector]
-      const account = prefix + adjacentConnector
-      return this.core.getPlugin(prefix).sendMessage({
-        ledger: prefix,
+    return Promise.all(adjacentLedgers.map((adjacentLedger) => {
+      return this._broadcastToLedger(adjacentLedger,
+        routes.filter((route) => route.source_ledger === adjacentLedger))
+    }))
+  }
+
+  _broadcastToLedger (adjacentLedger, routes) {
+    const connectors = Object.keys(this.peersByLedger[adjacentLedger])
+    return Promise.all(connectors.map((adjacentConnector) => {
+      const account = adjacentLedger + adjacentConnector
+      log.info('broadcasting ' + routes.length + ' routes to ' + account)
+      return this.core.getPlugin(adjacentLedger).sendMessage({
```

Comment:
For quoting there is https://github.com/interledger/rfcs/blob/master/0008-interledger-quoting-protocol/0008-interledger-quoting-protocol.md, but there's no spec for route broadcasting yet.